host_reference/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+include_directories(../utils)
+
+# enable utils build a common library for all utils
+set(QUDA_REFERENCE_COMMON covdev_reference.cpp dslash_test_helpers.cpp hisq_force_reference.cpp 
+  dslash_reference.cpp wilson_dslash_reference.cpp clover_reference.cpp domain_wall_dslash_reference.cpp 
+  gauge_force_reference.cpp hisq_force_reference2.cpp staggered_dslash_reference.cpp)
+cuda_add_library(quda_reference STATIC ${QUDA_REFERENCE_COMMON})
```

Comment:
can we use `add_library` here please? We are trying to get rid of the `cuda_` prefixed things as they are deprecated and actually currently a wrapper. And this should not have any cuda code in . Also I don't think we need QUDA_REFERENCE_COMMON here but can just pass the source directly to add_library.